52/kaijuking/pomodoro.py

```diff
@@ -0,0 +1,146 @@
+from datetime import datetime, timedelta
+import PySimpleGUI as sg
+
+
+def pomodoro_gui():
```

Comment:
Which so much logic embedded and possible need to keep state I would make this a class